homeassistant/components/hassio/__init__.py

```diff
@@ -130,7 +131,29 @@ def get_homeassistant_version(hass):
 
     Async friendly.
     """
-    return hass.data.get(DATA_HOMEASSISTANT_VERSION)
+    if DATA_INFO not in hass.data:
```

Comment:
Should we do this? I think it would be better to make it fail as it's an unexpected situation. Integrations should depend on hassio if they want to call its methods.

Reply:
I know... right now it uses `.get()` so I assumed it is expected to be `None` at some point. I wanted to keep that type of "risk" out of the loop now.

We should look at the integration anyways, as some things changed at the Supervisor (new endpoint and variables) that still need to be adjusted.